Guardfile

```diff
@@ -0,0 +1,55 @@
+# Defines the matching rules for Guard.
+guard :minitest, spring: true, all_on_start: false do
+  watch(%r{^test/(.*)/?(.*)_test\.rb$})
+  watch('test/test_helper.rb') { 'test' }
+  watch('config/routes.rb')    { integration_tests }
+  watch(%r{^app/models/(.*?)\.rb$}) do |matches|
+    "test/models/#{matches[1]}_test.rb"
+  end
+  watch(%r{^app/controllers/(.*?)_controller\.rb$}) do |matches|
+    resource_tests(matches[1])
+  end
+  watch(%r{^app/views/([^/]*?)/.*\.html\.erb$}) do |matches|
+    ["test/controllers/#{matches[1]}_controller_test.rb"] +
+    integration_tests(matches[1])
+  end
+  watch(%r{^app/helpers/(.*?)_helper\.rb$}) do |matches|
+    integration_tests(matches[1])
+  end
+  watch('app/views/layouts/application.html.erb') do
+    'test/integration/site_layout_test.rb'
+  end
+  watch('app/helpers/sessions_helper.rb') do
+    integration_tests << 'test/helpers/sessions_helper_test.rb'
+  end
+  watch('app/controllers/sessions_controller.rb') do
+    ['test/controllers/sessions_controller_test.rb',
+     'test/integration/users_login_test.rb']
+  end
+  watch('app/controllers/account_activations_controller.rb') do
+    'test/integration/users_signup_test.rb'
+  end
+  watch(%r{app/views/users/*}) do
+    resource_tests('users') +
+    ['test/integration/microposts_interface_test.rb']
+  end
+end
+
+# Returns the integration tests corresponding to the given resource.
+def integration_tests(resource = :all)
+  if resource == :all
+    Dir["test/integration/*"]
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)